data/json/itemgroups/SUS/domestic.json

```diff
@@ -651,16 +651,19 @@
     "entries": [
       {
         "distribution": [
-          { "item": "aspirin", "prob": 160, "charges": [ 1, 20 ] },
-          { "item": "codeine", "prob": 30, "charges": [ 1, 10 ] },
-          { "item": "tramadol", "prob": 10, "charges": [ 1, 10 ] }
+          { "item": "aspirin", "prob": 160, "charges": [ 1, 30 ] },
+          { "item": "codeine", "prob": 30, "charges": [ 1, 15 ] },
+          { "item": "tramadol", "prob": 10, "charges": [ 1, 15 ] }
         ],
         "prob": 95
       },
       { "item": "comb_lice", "prob": 10 },
       { "item": "eyedrops", "prob": 15 },
       { "item": "pepto", "prob": 70, "charges-min": 1, "container-item": "bottle_plastic_small" },
       { "item": "inhaler", "prob": 25, "charges-min": 10, "charges-max": 100 },
+      { "item": "disinfectant", "prob": 25, "charges-min": 1 },
```

Comment:
I would suggest putting disinfectand and hydrogen peroxide into their own distribution, with a probability of 40. Probably most bathrooms will have one or the other, not both.

Reply:
I had the same idea